troposphere/static/js/components/images/detail/header/HeaderView.react.js

```diff
@@ -22,6 +23,14 @@ export default React.createClass({
         });
     },
 
+    showLaunchModal: function(e) {
+        modals.InstanceModals.launch({
+            image: this.props.image,
+            initialView: "BASIC_VIEW"
+        });
+        trackAction("launched-from-image-detail", {});
```

Comment:
Can you comment on this? What is this method? Why are we passing an empty object, it's unlike js. We should pass no argument if it's optional.